src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to make new metrics available in the Observability Platform in self-service.
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-metrics
+weight: 50
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is a ServiceMonitor?
+  - What is a PodMonitor?
+  - How do I ingest new metrics?
+  - Why do my clusters run the Prometheus Operator?
+---
+
+By default, all Giant Swarm clusters are equiped with the [Prometheus Operator](https://prometheus-operator.dev/) and a set of multiple [Prometheus](https://prometheus.io/) shards in `agent mode` functioning as monitoring agents to be able to collect and forward cluster critical and workload metrics to a central [Grafana Mimir](https://grafana.com/oss/mimir/) instance running on the management cluster.
+
+No workload is the same, especially in the way it exposes its metrics, so the Observability Platforms monitoring configuration needs to be flexible. That's why it is based on the `ServiceMonitor` and `PodMonitor` Custom Resource Definitions (CRDs) provided by the Prometheus Operator. 
+
+Those allow you to:
+
+- Define where the metrics you want to ingest are (for example the container or port)
+- Transform metrics before ingesting them (for example dropping unneeded data, adding extra labels)
```

Comment:
```suggestion
- Transform metrics before ingesting them (i.e dropping unneeded data, adding extra labels...)
```
Feel free to ignore this one, I just find it more elegant this way

Reply:
Well, me to but validators don't agree